### PR TITLE
add partitioned raft kv page

### PR DESCRIPTION
**Pull request by tonyxuqqi**
Signed-off-by: tonyxuqqi <tonyxuqi@outlook.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
add partitioned-raft-kv.md

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by TomShawn** [comment]
@tonyxuqqi Please sign the [CLA](https://github.com/pingcap/docs-cn/pull/12978#issuecomment-1426637421) and invite one or more technical review.

File: partitioned_raft_kv.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+title: 分区raft kv
+aliases: ['/docs-cn/dev/partitioned-raft-kv/']
+---
+
+# 分区raft kv
+
+本文介绍 TiKV 的分区raft kv。
+
+## 介绍
+在v6.6以前的版本中, TiKV基于Raft的后台存储引擎使用一个单一的RocksDB实例。一个TiKV上所有Region的数据都存储在该RocksDB实例中。
```
Comment (benmaoer) [inline partitioned_raft_kv.md:11]:
```suggestion
TiDB v6.6.0 之前，TiKV 基于 Raft 的存储引擎使用一个单一的 RocksDB 实例存储该 TiKV 实例所有 Region 的数据。
```

File: partitioned_raft_kv.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+在v6.6中，我们引入了一个新的Raft存储引擎，该引擎仍然使用RocksDB，但每个Region的数据都存储在单独的RocksDB实例中，也就是说每个RocksDB也是分区的，这也是分区raft kv名字的由来。目前该功能是实验特性，不推荐在生产环境中使用。
```
Comment (benmaoer) [inline partitioned_raft_kv.md:12]:
```suggestion
为了更平稳的支持更大的集群，从 TiDB v6.6.0 开始，我们引入了一个全新的 TiKV 存储引擎，该引擎使用多个 RocksDB 实例来存储 TiKV 的 Region 数据，每个 Region 的数据都独立存储在单独的 RocksDB 实例中。新引擎能够更好的控制 RocksDB 实例的文件数和层级，并实现了 Region 间数据操作的物理隔离，避免互相影响，支持平稳管理更多的数据。可以理解为每个 RocksDB 实例是分区管理的，这也是该特性分区 Raft KV 名字的由来。该功能目前是实验特性，不推荐在生产环境中使用。
```

File: partitioned_raft_kv.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+该功能的主要优势在于更好的写入性能，更快的扩缩容，相同硬件下可以支持更大的数据，也能支持更大的集群规模。
+
+## 相关配置
+### storage.engine
+设置engine类型。该配置只能在创建新集群时指定，且后续无法更改。**实验特性**
+* `raft-kv`: raft-kv是6.6以前的Engine，也是该选项的默认值
```
Comment (benmaoer) [inline partitioned_raft_kv.md:18]:
```suggestion
* `raft-kv`: raft-kv是 TiDB v6.6.0 之前的默认存储引擎
```

File: partitioned_raft_kv.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+* `partitioned-raft-kv`: partitioned-raft-kv是6.6新引入的Engine类型。
```
Comment (benmaoer) [inline partitioned_raft_kv.md:19]:
```suggestion
* `partitioned-raft-kv`: partitioned-raft-kv 是 TiDB v6.6.0 新引入的存储引擎
```

File: partitioned_raft_kv.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+## 该功能的使用场景
+* 需要在单个TiKV支持更大的数据
+* 有大量写入吞吐
+* 频繁做扩缩容
+* 读、写放大比较严重的负载
+* TiKV内存尚有富余
+
+## 该功能的限制
+由于该功能处于实验特性，以下功能仍然在开发中，目前不支持。
+* lightning导入, TiCDC, BR, dumping, Tikv-ctl等工具均不支持
```
Comment (benmaoer) [inline partitioned_raft_kv.md:40]:
```suggestion
* 暂不支持 Lightning，TiCDC，BR，PITR，dumping 等数据导入、同步和备份工具
* 暂不支持 tikv-ctl 命令行管理工具
```

File: partitioned_raft_kv.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+该功能的主要优势在于更好的写入性能，更快的扩缩容，相同硬件下可以支持更大的数据，也能支持更大的集群规模。
+
+## 相关配置
+### storage.engine
+设置engine类型。该配置只能在创建新集群时指定，且后续无法更改。**实验特性**
+* `raft-kv`: raft-kv是6.6以前的Engine，也是该选项的默认值
+* `partitioned-raft-kv`: partitioned-raft-kv是6.6新引入的Engine类型。
+
+### rocksdb.write-buffer-limit
+设置单个TiKV中所有RocksDB中使用的memtable的内存上限,默认值为本机内存的25%, 推荐不低于5GB。该选项只作用于`partitioned-raft-kv`.**实验特性**
```
Comment (benmaoer) [inline partitioned_raft_kv.md:22]:
```suggestion
设置单个 TiKV 所有的 RocksDB 实例使用的 memtable 的总内存上限，默认值为本机内存的 25%，推荐配置不低于 5GB 的内存。该配置只作用于`partitioned-raft-kv`.**实验特性**
```

**Review by benmaoer — CHANGES_REQUESTED** [review]

**Review by benmaoer — APPROVED** [review]

File: partitioned_raft_kv.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
+---
+title: 分区 Raft KV
```
Comment (shichun-0415) [inline partitioned_raft_kv.md:2]: 请将这篇文档增加到 TOC.md，否则文档不会出现在左侧导航。

Comment (tonyxuqqi) [reply partitioned_raft_kv.md:2]: > 请将这篇文档增加到 TOC.md，否则文档不会出现在左侧导航。
有什么建议加到哪里吗？最好是类似What's new这样的一栏里

File: tikv-configuration-file.md [hunk 8]
```diff
@@ -1127,9 +1159,7 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
 ### `block-size`
 
 + 一个 RocksDB block 的默认大小。
-+ `defaultcf` 默认值：64KB
-+ `writecf` 默认值：64KB
-+ `lockcf` 默认值：16KB
++ 默认值：16KB
```
Comment (TomShawn) [inline tikv-configuration-file.md:1162]:
```suggestion
+ `defaultcf` 默认值：64KB
+ `writecf` 默认值：64KB
+ `lockcf` 默认值：16KB	
```

File: tikv-configuration-file.md [hunk 9]
```diff
@@ -1129,7 +1161,7 @@ rocksdb defaultcf、rocksdb writecf 和 rocksdb lockcf 相关的配置项。
 + 一个 RocksDB block 的默认大小。
 + `defaultcf` 默认值：64KB
 + `writecf` 默认值：64KB
-+ `lockcf` 默认值：16KB
++ `lockcf` 默认值：16KB
```
Comment (TomShawn) [inline tikv-configuration-file.md:1164]:
```suggestion
+ `lockcf` 默认值：16KB
```

File: partitioned-raft-kv.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
+---
+title: 分区 Raft KV
+summary: 了解 TiKV 的分区 Raft KV 特性。
+---
+
+# 分区 Raft KV
+
+## 概述
+
```
Comment (TomShawn) [inline partitioned-raft-kv.md:9]:
```suggestion
```

File: partitioned-raft-kv.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+> **警告：**
+>
+> 分区 Raft KV 目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
+v6.6.0 之前，基于 Raft 的存储引擎，TiKV 使用单一的 RocksDB 实例存储该 TiKV 实例所有 Region 的数据。
+
+为了更平稳地支持更大的集群，从 v6.6.0 开始，TiDB 引入了一个全新的 TiKV 存储引擎，该引擎使用多个 RocksDB 实例来存储 TiKV 的 Region 数据，每个 Region 的数据都独立存储在单个 RocksDB 实例中。
+
+## 原理及优势
```
Comment (TomShawn) [inline partitioned-raft-kv.md:18]:
```suggestion
```

File: partitioned-raft-kv.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+
+新的 TiKV 引擎能够更好地控制 RocksDB 实例的文件数和层级，并实现了 Region 间数据操作的物理隔离，避免相互影响。同时，该引擎支持平稳管理更多的数据。你可以理解为，TiKV 通过分区管理多个 RocksDB 实例，这也是该特性分区 Raft KV 名字的由来。
+
```
Comment (TomShawn) [inline partitioned-raft-kv.md:21]: 或者将使用方法放到使用场景下面？
```suggestion

## 使用方法

```

File: partitioned-raft-kv.md [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+* 需要在单个 TiKV 支持更多的数据。
+* 有大量写入吞吐。
+* 需要频繁地扩缩容。
+* 负载有较为严重的读写放大。
+* TiKV 内存尚有富余。
+
+## 使用方法
+
+要启用 Raft KV，需要将配置项 [`storage.engine`](/tikv-configuration-file.md#storageengine-从-v660-版本开始引入) 设为 `raft-kv`。同时，可以在使用 Raft KV 时，通过配置项 [`rocksdb.write-buffer-flush-oldest-first`](/tikv-configuration-file.md#rocksdbwrite-buffer-flush-oldest-first-从-v660-版本开始引入) 和 [`rocksdb.write-buffer-limit`](/tikv-configuration-file.md#rocksdbwrite-buffer-limit-从-v660-版本开始引入) 来控制 RocksDB 的内存使用。
```
Comment (BusyJay) [inline partitioned-raft-kv.md:30]: s/raft-kv/partitioned-raft-kv

Comment (shichun-0415) [reply partitioned-raft-kv.md]: Done in [878d7ec](https://github.com/pingcap/docs-cn/pull/12978/commits/878d7ec9ed48a9ffd2606566a27c435f9e4dd94b)

File: tikv-configuration-file.md [hunk 14]
```diff
@@ -437,6 +437,17 @@ RocksDB 多个 CF 之间共享 block cache 的配置选项。
...
+## storage.engine <span class="version-mark">从 v6.6.0 版本开始引入</span>
+
+> **警告：**
+>
+> 该功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
+设置 engine 类型，可选值有 `raft-kv` 和 `partitioned-raft-kv`。该配置只能在创建新集群时指定，且后续无法更改。
+
+- `raft-kv`：TiDB v6.6.0 之前的默认存储引擎。
+- `partitioned-raft-kv`：TiDB v6.6.0 新引入的存储引擎。
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:449]:
```suggestion
+ 设置存储引擎类型。该配置只能在创建新集群时指定，且后续无法更改。
+ 可选值：

    + `raft-kv`：TiDB v6.6.0 之前的默认存储引擎。
    + `partitioned-raft-kv`：TiDB v6.6.0 新引入的存储引擎。
```

File: tikv-configuration-file.md [hunk 15]
```diff
@@ -985,6 +996,27 @@ rocksdb 相关的配置项。
...
+### rocksdb.write-buffer-flush-oldest-first <span class="version-mark">从 v6.6.0 版本开始引入</span>
+
+> **警告：**
+>
+> 该功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
+设置当 RocksDB 当前 memtable 内存占用达到阈值之后的 Flush 策略。
+
+- `false`：默认值，Flush 策略是优先选择数据量大的 memtable 落盘到 SST。
+- `true`：Flush 策略是优先选择最早的 memtable 落盘到 SST。该策略可以清除冷数据的 memtable，用于有明显冷热数据的场景。
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1008]:
```suggestion
+ 设置当 RocksDB 当前 memtable 内存占用达到阈值之后的 Flush 策略。
+ 默认值：`false`
+ 可选值：
    + `false`：Flush 策略是优先选择数据量大的 memtable 落盘到 SST。
    + `true`：Flush 策略是优先选择最早的 memtable 落盘到 SST。该策略可以清除冷数据的 memtable，用于有明显冷热数据的场景。
```

File: partitioned-raft-kv.md [hunk 16]
```diff
@@ -0,0 +1,41 @@
...
+summary: 了解 TiKV 的分区 Raft KV 特性。
+---
+
+# 分区 Raft KV
+
+> **警告：**
+>
+> 分区 Raft KV 目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
+v6.6.0 之前，基于 Raft 的存储引擎，TiKV 使用单一的 RocksDB 实例存储该 TiKV 实例所有 Region 的数据。
```
Comment (Oreoxmt) [inline partitioned-raft-kv.md:12]:
```suggestion
v6.6.0 之前，基于 Raft 的存储引擎，TiKV 使用单个 RocksDB 实例存储该 TiKV 实例所有 Region 的数据。
```

File: partitioned-raft-kv.md [hunk 17]
```diff
@@ -0,0 +1,41 @@
...
+
+为了更平稳地支持更大的集群，从 v6.6.0 开始，TiDB 引入了一个全新的 TiKV 存储引擎，该引擎使用多个 RocksDB 实例来存储 TiKV 的 Region 数据，每个 Region 的数据都独立存储在单个 RocksDB 实例中。
+
+新的 TiKV 引擎能够更好地控制 RocksDB 实例的文件数和层级，并实现了 Region 间数据操作的物理隔离，避免相互影响。同时，该引擎支持平稳管理更多的数据。你可以理解为，TiKV 通过分区管理多个 RocksDB 实例，这也是该特性分区 Raft KV 名字的由来。
+
+## 使用场景
+
+如果你的 TiKV 集群有以下特点，可以考虑使用该功能：
+
+* 需要在单个 TiKV 支持更多的数据。
```
Comment (Oreoxmt) [inline partitioned-raft-kv.md:22]:
```suggestion
* 需要在单个 TiKV 实例支持更多的数据。
```

File: partitioned-raft-kv.md [hunk 18]
```diff
@@ -0,0 +1,41 @@
...
+* 有大量写入吞吐。
```
Comment (Oreoxmt) [inline partitioned-raft-kv.md:23]:
```suggestion
* 具有大量写入吞吐。
```

File: partitioned-raft-kv.md [hunk 19]
```diff
@@ -0,0 +1,41 @@
...
+* 需要频繁地扩缩容。
+* 负载有较为严重的读写放大。
+* TiKV 内存尚有富余。
+
+该功能的主要优势在于更好的写入性能，更快的扩缩容，相同硬件下可以支持更大的数据，也能支持更大的集群规模。
```
Comment (Oreoxmt) [inline partitioned-raft-kv.md:28]:
```suggestion
该功能的主要优势在于提高写入性能、加快扩缩容速度以及在相同硬件下可以支持更大的数据和更大的集群规模。
```

File: partitioned-raft-kv.md [hunk 20]
```diff
@@ -0,0 +1,41 @@
...
+
+## 使用方法
+
+要启用 Raft KV，需要在创建集群时将配置项 [`storage.engine`](/tikv-configuration-file.md#storageengine-从-v660-版本开始引入) 设为 `partitioned-raft-kv`。同时，可以在使用 Raft KV 时，通过配置项 [`rocksdb.write-buffer-flush-oldest-first`](/tikv-configuration-file.md#rocksdbwrite-buffer-flush-oldest-first-从-v660-版本开始引入) 和 [`rocksdb.write-buffer-limit`](/tikv-configuration-file.md#rocksdbwrite-buffer-limit-从-v660-版本开始引入) 来控制 RocksDB 的内存使用。
```
Comment (Oreoxmt) [inline partitioned-raft-kv.md:32]:
```suggestion
要启用 Raft KV，需要在创建集群时将配置项 [`storage.engine`](/tikv-configuration-file.md#storageengine-从-v660-版本开始引入) 设为 `partitioned-raft-kv`。同时，在使用 Raft KV 时，可以通过配置项 [`rocksdb.write-buffer-flush-oldest-first`](/tikv-configuration-file.md#rocksdbwrite-buffer-flush-oldest-first-从-v660-版本开始引入) 和 [`rocksdb.write-buffer-limit`](/tikv-configuration-file.md#rocksdbwrite-buffer-limit-从-v660-版本开始引入) 来控制 RocksDB 的内存使用。
```

File: partitioned-raft-kv.md [hunk 21]
```diff
@@ -0,0 +1,41 @@
...
+
+## 使用限制
+
+由于该功能为实验特性，目前有以下限制：
+
+* 暂不支持 TiDB Lightning、TiCDC、BR、Dumping 等数据导入、同步和备份工具。
+* 暂不支持 tikv-ctl 命令行管理工具。
+* 不支持同时和 TiFlash 使用。
+* 需要在创建集群时启用，不支持集群创建后开启。
```
Comment (Oreoxmt) [inline partitioned-raft-kv.md:41]:
```suggestion
* 需要在创建集群时启用，不支持在集群创建后开启。
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

File: tikv-configuration-file.md [hunk 22]
```diff
@@ -437,6 +437,18 @@ RocksDB 多个 CF 之间共享 block cache 的配置选项。
...
+
+> **警告：**
+>
+> 该功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
++ 设置存储引擎类型。该配置只能在创建新集群时指定，且后续无法更改。
++ 可选值：
+
+    + `raft-kv`：TiDB v6.6.0 之前的默认存储引擎。
+    + `partitioned-raft-kv`：TiDB v6.6.0 新引入的存储引擎。
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:450]:
```suggestion
    + `"raft-kv"`：TiDB v6.6.0 之前的默认存储引擎。
    + `"partitioned-raft-kv"`：TiDB v6.6.0 新引入的存储引擎。
```

File: releases/release-6.6.0.md [hunk 23]
```diff
@@ -386,6 +388,9 @@ TiDB 版本：6.6.0-[DMR](/releases/versioning.md#开发里程碑版本)
 | TiDB | [`tidb_stmt_summary_file_max_size`](/tidb-configuration-file.md#tidb_stmt_summary_file_max_size-从-v660-版本开始引入) | 新增 | 当开启了 statements summary 持久化时，该配置用于限制持久化数据单个文件的大小 (MiB)，默认值为 `64`。 |
 | TiDB | [`tidb_stmt_summary_filename`](/tidb-configuration-file.md#tidb_stmt_summary_filename-从-v660-版本开始引入) | 新增 | 当开启了 statements summary 持久化时，该配置用于指定持久化数据所写入的文件名称，默认为 `tidb-statements.log`。 |
 | TiKV | [`resource-control.enabled`](/tikv-configuration-file.md#resource-control) | 新增 | 控制是否支持对用户前台的读写请求按照对应的资源组配额做优先级调度。默认为 `false`，即关闭按照资源组配额调度。 |
+| TiKV | [`storage.engine`](/tikv-configuration-file.md#storageengine-从-v660-版本开始引入) | 新增 | 用于设置 engine 类型，可选值有 `raft-kv` 和 `partitioned-raft-kv`。该配置只能在创建新集群时指定，且后续无法更改。 |
```
Comment (Oreoxmt) [inline releases/release-6.6.0.md:391]:
```suggestion
| TiKV | [`storage.engine`](/tikv-configuration-file.md#storageengine-从-v660-版本开始引入) | 新增 | 用于设置存储引擎类型，可选值有 `"raft-kv"` 和 `"partitioned-raft-kv"`。该配置只能在创建新集群时指定，且后续无法更改。 |
```

File: tikv-configuration-file.md [hunk 24]
```diff
@@ -985,6 +997,28 @@ rocksdb 相关的配置项。
...
+    + `false`：Flush 策略是优先选择数据量大的 memtable 落盘到 SST。
+    + `true`：Flush 策略是优先选择最早的 memtable 落盘到 SST。该策略可以清除冷数据的 memtable，用于有明显冷热数据的场景。
+
+### rocksdb.write-buffer-limit <span class="version-mark">从 v6.6.0 版本开始引入</span>
+
+> **警告：**
+>
+> 该功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
++ 设置单个 TiKV 中所有 RocksDB 实例使用的 memtable 的总内存上限，默认值为本机内存的 25%，推荐配置不低于 5 GiB 的内存。该配置只对 `partitioned-raft-kv` 生效。
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1018]:
```suggestion
+ 设置单个 TiKV 中所有 RocksDB 实例使用的 memtable 的总内存上限，默认值为本机内存的 25%，推荐配置不低于 5 GiB 的内存。该配置只对分区 Raft KV (storage.engine="partitioned-raft-kv") 生效。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: tikv-configuration-file.md [hunk 25]
```diff
@@ -985,6 +997,28 @@ rocksdb 相关的配置项。
 + 自动创建 DB 开关。
 + 默认值：true
 
+### rocksdb.write-buffer-flush-oldest-first <span class="version-mark">从 v6.6.0 版本开始引入</span>
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1001]:
```suggestion
### `write-buffer-flush-oldest-first` <span class="version-mark">从 v6.6.0 版本开始引入</span>
```

File: tikv-configuration-file.md [hunk 26]
```diff
@@ -985,6 +997,28 @@ rocksdb 相关的配置项。
...
+>
+> 该功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
++ 设置当 RocksDB 当前 memtable 内存占用达到阈值之后的 Flush 策略。
++ 默认值：`false`
++ 可选值：
+    + `false`：Flush 策略是优先选择数据量大的 memtable 落盘到 SST。
+    + `true`：Flush 策略是优先选择最早的 memtable 落盘到 SST。该策略可以清除冷数据的 memtable，用于有明显冷热数据的场景。
+
+### rocksdb.write-buffer-limit <span class="version-mark">从 v6.6.0 版本开始引入</span>
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1013]:
```suggestion
### `write-buffer-limit` <span class="version-mark">从 v6.6.0 版本开始引入</span>
```

File: tikv-configuration-file.md [hunk 27]
```diff
@@ -437,6 +437,18 @@ RocksDB 多个 CF 之间共享 block cache 的配置选项。
 + 默认值：系统总内存大小的 45%
 + 单位：KB|MB|GB
 
+## storage.engine <span class="version-mark">从 v6.6.0 版本开始引入</span>
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:440]:
```suggestion
## `engine` <span class="version-mark">从 v6.6.0 版本开始引入</span>
```

Move this under L358 https://docs.pingcap.com/tidb/dev/tikv-configuration-file#storage

Comment (shichun-0415) [reply tikv-configuration-file.md]: Done in [63c02c0](https://github.com/pingcap/docs-cn/pull/12978/commits/63c02c0ef19cd73842408ef9d1508899466e208f)

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: tikv-configuration-file.md [hunk 28]
```diff
@@ -1098,6 +1116,28 @@ rocksdb 相关的配置项。
...
+>
+> 该功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
++ 设置当 RocksDB 当前 memtable 内存占用达到阈值之后的 Flush 策略。
++ 默认值：`false`
++ 可选值：
+    + `false`：Flush 策略是优先选择数据量大的 memtable 落盘到 SST。
+    + `true`：Flush 策略是优先选择最早的 memtable 落盘到 SST。该策略可以清除冷数据的 memtable，用于有明显冷热数据的场景。
+
+### write-buffer-limit <span class="version-mark">从 v6.6.0 版本开始引入</span>
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1131]:
```suggestion
### `write-buffer-limit` <span class="version-mark">从 v6.6.0 版本开始引入</span>
```

File: tikv-configuration-file.md [hunk 29]
```diff
@@ -1098,6 +1116,28 @@ rocksdb 相关的配置项。
 + 日志存储目录。
 + 默认值：""
 
+### write-buffer-flush-oldest-first <span class="version-mark">从 v6.6.0 版本开始引入</span>
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:1119]:
```suggestion
### `write-buffer-flush-oldest-first` <span class="version-mark">从 v6.6.0 版本开始引入</span>
```

File: tikv-configuration-file.md [hunk 30]
```diff
@@ -359,6 +359,24 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 
 存储相关的配置项。
 
+### data dir
+
++ RocksDB 存储路径。
++ 默认值：`"./"`
+
+### engine <span class="version-mark">从 v6.6.0 版本开始引入</span>
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:367]:
```suggestion
### `engine` <span class="version-mark">从 v6.6.0 版本开始引入</span>
```

File: tikv-configuration-file.md [hunk 31]
```diff
@@ -359,6 +359,24 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 
 存储相关的配置项。
 
+### data dir
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:362]:
```suggestion
### `data-dir`
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: tikv-configuration-file.md (hunk 31 above)
Comment (BusyJay) [inline tikv-configuration-file.md:362]: 这个配置一直都有的吧

Comment (shichun-0415) [reply tikv-configuration-file.md:362]: 是的，但是发现英文有，中文没有，所以加上了。

File: partitioned-raft-kv.md [hunk 32]
```diff
@@ -0,0 +1,41 @@
...
+* 具有大量写入吞吐。
+* 需要频繁地扩缩容。
+* 负载有较为严重的读写放大。
+* TiKV 内存尚有富余。
+
+该功能的主要优势在于，提高写入性能，加快扩缩容速度，以及在相同硬件下支持更多数据和更大的集群。
+
+## 使用方法
+
+要启用 Raft KV，需要在创建集群时将配置项 [`storage.engine`](/tikv-configuration-file.md#engine-从-v660-版本开始引入) 设为 `"partitioned-raft-kv"`。同时，在使用 Raft KV 时，可以通过配置项 [`rocksdb.write-buffer-flush-oldest-first`](/tikv-configuration-file.md#write-buffer-flush-oldest-first-从-v660-版本开始引入) 和 [`rocksdb.write-buffer-limit`](/tikv-configuration-file.md#write-buffer-limit-从-v660-版本开始引入) 来控制 RocksDB 的内存使用。
```
Comment (benmaoer) [inline partitioned-raft-kv.md:32]:
```suggestion
要启用分区 Raft KV，需要在创建集群时将配置项 [`storage.engine`](/tikv-configuration-file.md#engine-从-v660-版本开始引入) 设为 `"partitioned-raft-kv"`。同时，在使用分区 Raft KV 特性时，可以通过配置项 [`rocksdb.write-buffer-flush-oldest-first`](/tikv-configuration-file.md#write-buffer-flush-oldest-first-从-v660-版本开始引入) 和 [`rocksdb.write-buffer-limit`](/tikv-configuration-file.md#write-buffer-limit-从-v660-版本开始引入) 来控制 RocksDB 的内存使用。
```

**Review by benmaoer — CHANGES_REQUESTED** [review]

**Review by benmaoer — APPROVED** [review]
LGTM

**Pull request merged**